crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__expression.py.snap

```diff
@@ -331,24 +331,19 @@ last_call()
  [i for i in (1, 2, 3)]
  [(i**2) for i in (1, 2, 3)]
  [(i**2) for i, _ in ((1, "a"), (2, "b"), (3, "c"))]
- [((i**2) + j) for i in (1, 2, 3) for j in (1, 2, 3)]
--{i: 0 for i in (1, 2, 3)}
--{i: j for i, j in ((1, "a"), (2, "b"), (3, "c"))}
--{a: b * 2 for a, b in dictionary.items()}
--{a: b * -2 for a, b in dictionary.items()}
--{
--    k: v
+@@ -101,7 +104,10 @@
+ {a: b * -2 for a, b in dictionary.items()}
+ {
+     k: v
 -    for k, v in this_is_a_very_long_variable_which_will_cause_a_trailing_comma_which_breaks_the_comprehension
--}
-+{NOT_IMPLEMENTED_dict_key: NOT_IMPLEMENTED_dict_value for key, value in NOT_IMPLEMENTED_dict}
-+{NOT_IMPLEMENTED_dict_key: NOT_IMPLEMENTED_dict_value for key, value in NOT_IMPLEMENTED_dict}
-+{NOT_IMPLEMENTED_dict_key: NOT_IMPLEMENTED_dict_value for key, value in NOT_IMPLEMENTED_dict}
-+{NOT_IMPLEMENTED_dict_key: NOT_IMPLEMENTED_dict_value for key, value in NOT_IMPLEMENTED_dict}
-+{NOT_IMPLEMENTED_dict_key: NOT_IMPLEMENTED_dict_value for key, value in NOT_IMPLEMENTED_dict}
++    for (
++        k,
++        v,
++    ) in this_is_a_very_long_variable_which_will_cause_a_trailing_comma_which_breaks_the_comprehension
```

Comment:
Note the generators formatting.

Reply:
cc: @davidszotten

Reply:
You mean the parentheses around the tuple and that it expands over multiple lines? We probably need to use the `StripInsideForLoop` layout here as well (unrelated to your PR)

https://github.com/astral-sh/ruff/issues/5779